.github/workflows/main.yml

```
@@ -0,0 +1,44 @@
+name: Singularity container build
+
+on:
+  push:
+    branches: [ master,ghaction ]
```

```suggestion
    branches: [ master ]
```
need to commit before merging